### PR TITLE
[GeckoView] Add missing CSS variables for the dialog functionality

### PR DESCRIPTION
**Pull request by Snuffleupagus**

### CODE REVIEW

File: web/app.js [hunk 1]
```diff
@@ -1005,7 +1005,10 @@ const PDFViewerApplication = {
     this._saveInProgress = false;
     this._hasAnnotationEditors = false;
 
-    promises.push(this.pdfScriptingManager.destroyPromise);
+    promises.push(
+      this.pdfScriptingManager.destroyPromise,
+      this.passwordPrompt.close()
```
Comment (calixteman) [inline web/app.js:1010]: It doesn't seem to be related to what the title of the PR is about.

**Comment by calixteman** [comment]
Is a dialog viewable in Fenix ?

File: web/app.js (hunk 1 above)
Comment (Snuffleupagus) [reply web/app.js:1010]: That's why this was done in *separate* commit, since the change is small enough to not really warrant a separate PR in my opinion.

**Comment by Snuffleupagus** [comment]
> Is a dialog viewable in Fenix ?

Barely, this is how the GV development-viewer looks on `master` when opening a password protected PDF document:

master

This is how it looks with this patch, which as the commit message says isn't perfect but still a lot more usable:

patch

**Review by calixteman — APPROVED** [review]
LGTM. Thank you.
